services/py-api/src/database/repository/participants_repository.py

```diff
@@ -48,12 +46,29 @@ async def create(
                 # If the team_id is passed as a kwarg the participant will be inserted in the given team
                 team_id=kwargs.get("team_id"),
             )
-            LOG.debug("Inserting participant...", participant=participant.dump_as_json())
+            if session:
+                LOG.debug(
+                    "Inserting participant {participant} via transaction {id}",
```

Comment:
You do not need the things in the {}, they are not replaced with the values, they just sit there as strings